content/en/account_management/audit_logs.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Audit Logs
+kind: documentation
+further_reading:
+- link: "/account_management/org_settings/"
+  tag: "Documentation"
+  text: "Learn more about organization settings"
+---
+
+<div class="alert alert-warning">This feature is in public beta. Contact <a href="https://www.datadoghq.com/support/">Datadog support</a> for more information.</div>
+
+## Overview
+
+As an administrator or security team member, you can see who is using Datadog within your organization and the context in which they are using Datadog with [Audit Logs][1]. As an individual, you can see a stream of your own actions, too.
+
+There are two types of events that can occur within an audit log:
+
+1. Request events, which translate all requests made to Datadog’s API into a customer record
+2. Product-specific events, such as:
+    - What has changed within your organization or account. For example, if a user changes a dashboard, you are able to see who made the last change and at what time.
+    - Who accessed and viewed certain information within your environment, for compliance audits.
+    - A breaking event, such as seeing what API calls led up to the event.
```

Comment:
this makes it seem like "seeing what api calls led up to the event" is an example of a breaking event

```suggestion
    - Breaking events. In these cases, you can see what API calls led up to the event.
```